translations/README/MALYALAM.md

```diff
@@ -0,0 +1,370 @@
+# [ഈ പ്രോജക്ടിൽ സംഭാവന ചെയ്യുക](https://syknapse.github.io/Contribute-To-This-Project/)
+
+![ചിത്ര വിവരം](/favicon.png)
+
+> ലോഗോ :sparkling_heart: കൊണ്ടു സൃഷ്ടിച്ചത് [CandidDeer](https://github.com/CandidDeer)
+
+[![ട്വീറ്റ്](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
+
+[![ഡിസ്കോർഡ്](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Join%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'നമ്മുടെ ഡിസ്കോർഡ് സർവറിൽ ചേരുക!')
+[![PRs സ്വാഗതം](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Project/)
+[![ഓപ്പൺ സോഴ്‌സ് സ്നേഹം](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Contribute-To-This-Project/)
+
+---
+
+> ## **അറിയിപ്പ്:**
+>
+> നിങ്ങൾ ഈ പ്രോജക്റ്റിൽ ഒരു മെയിന്റെയ്‌നറായി തുടരണം, അത് തുടരാൻ സഹായിക്കണോ? നിങ്ങൾക്ക് താത്പര്യമുണ്ടെങ്കിൽ, [മെയിന്റെയ്‌നറിന്റെ ഗൈഡ്](/maintainer_guide.md) വായിക്കുക, ഞങ്ങളുടെ [ഡിസ്കോർഡ് സർവറിൽ](https://discord.gg/tWkvS4ueVF) ചേരുക, പ്രോജക്ട് മെയിന്റെയ്‌നർമാരിൽ നിന്ന് ടീമിൽ ചേരാൻ അഭ്യർത്ഥിക്കുക.
+
+---
+
+### ത്വരിത ആക്സസ് സൂചിക
+
+- [പരിചയം](#introduction)
+- [ലക്ഷ്യങ്ങൾ](#objectives)
+- [ഇത് ആരാണ് വേണ്ടത്?](#who-is-this-for)
+- [എനിക്ക് ഇത് ചെയ്യേണ്ടത് എന്തിന്?](#why-do-i-need-to-do-this)
+- [ഞാൻ എന്ത് സംഭാവന ചെയ്യാൻ പോകുന്നു?](#what-am-i-going-to-contribute)
+- [മൊഴിമാറ്റങ്ങൾ](#translations)
+- [സജ്ജീകരണം! :)](#setup-)
+- [സംഭാവന ചെയ്യുക](#contribute)
+  - [പടിവാതിൽ 1: ഈ റിപ്പോസിറ്ററി Fork ചെയ്യുക](#step-1-fork-this-repository)
+  - [പടിവാതിൽ 2: റിപ്പോസിറ്ററി ക്ലോൺ ചെയ്യുക](#step-2-clone-the-repository)
+  - [പടിവാതിൽ 3: പുതിയ ഒരു ബ്രാഞ്ച് സൃഷ്ടിക്കുക](#step-3-create-a-new-branch)
+  - [പടിവാതിൽ 4: index.html ഫയൽ തുറക്കുക](#step-4-open-the-indexhtml-file)
+  - [പടിവാതിൽ 5: കാർഡ് ടെംപ്ലേറ്റ് പകർത്തുക](#step-5-copy-the-card-template)
+  - [പടിവാതിൽ 6: നിങ്ങളുടെ മാറ്റങ്ങൾ പ്രയോഗിക്കുക](#step-6-apply-your-changes)
+  - [പടിവാതിൽ 7: നിങ്ങളുടെ മാറ്റങ്ങൾ Commit ചെയ്യുക](#step-7-commit-your-changes)
+  - [പടിവാതിൽ 8: GitHub-ൽ നിങ്ങളുടെ മാറ്റങ്ങൾ Push ചെയ്യുക](#step-8-push-your-changes-to-github)
+  - [പടിവാതിൽ 9: PR (പൂൾ റിക്വസ്റ്റ്) സമർപ്പിക്കുക](#step-9-submit-a-prpull-request)
+  - [പടിവാതിൽ 10: ആഘോഷിക്കുക](#step-10-celebrate)
+- [അടുത്ത ചുവടുകൾ](#next-steps)
+- [അഭിനന്ദനങ്ങൾ](#acknowledgements)
+- [ലൈസൻസ്](#license)
+- [മുകളിൽ 100 സംഭാവനക്കാർ](#top-100-contributors)
```

Comment:
the url of this content layout table is still messy, kindly fix it.

Reply:
On it.